lib/actions/apiV2.js

```diff
@@ -1203,27 +1203,36 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
                 response.data?.plan?.routingErrors.filter(
                   (re) => re?.code !== 'NO_TRANSIT_CONNECTION'
                 )
-              if (
-                getActiveItineraries(getState())?.length > 0 &&
-                response?.data?.plan
-              ) {
-                // Add accessibility error, if it is turned on
+              if (withCollapsedShortNames.length > 0 && response?.data?.plan) {
+                if (!response.data.plan?.routingErrors) {
+                  response.data.plan.routingErrors = []
+                }
+                // Add configurable errors, if they're turned on.
                 const state = getState()
-                const { displayA11yError } = state.otp.config?.itinerary
+                const { displayA11yError, displayMicromobilityRentalWarning } =
+                  state.otp.config?.itinerary
                 if (displayA11yError) {
                   if (
-                    response.data.plan.itineraries.find(
+                    withCollapsedShortNames.find(
                       (itin) => !!itin?.accessibilityScore
                     )
                   ) {
-                    if (!response.data.plan?.routingErrors) {
-                      response.data.plan.routingErrors = []
-                    }
                     response.data.plan.routingErrors.push({
                       code: 'OTP_RR_A11Y_ROUTING_ENABLED'
                     })
                   }
                 }
+                if (displayMicromobilityRentalWarning) {
+                  if (
+                    withCollapsedShortNames.find((itin) =>
+                      itin.legs.find((leg) => leg.rentedBike)
```

Comment:
Can we add a comment that this is a deprecated field and we should change it?

Reply:
I'm not seeing anything to show this is deprecated? It's not noted here whereas other fields are: https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/types/Leg